kitchen-vagrant.gemspec

```diff
@@ -16,6 +16,4 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
 
   gem.required_ruby_version = ">= 3.0"
-
-  gem.add_dependency "test-kitchen", ">= 1.4", "< 4"
```

Comment:
Could this maybe be worked around on your environment with an environment variable? Perhaps something like this:
```ruby
unless ENV['CHEF_TEST_KITCHEN_ENTERPRISE']
  gem.add_dependency "test-kitchen", ">= 1.4", "< 4"
end
```
I'm not sure if this will work but it's a thought. By default I'd like to keep this dependency if possible.

Will the other gem be published on rubygems.org or only in a private repo?

Reply:
didn't think of that. Then could do if statement ENV true gem ctke else tk. Does that mean though that whever the vagrant gem is used with ctke that they'd need to set that attribute to true or only when doing something like gem or bundle install?